active/website.md

```diff
@@ -0,0 +1,61 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Ensure that information on the website is accurate 
+- Help to make the website accessible to all
+
+
+### Delegated responsibilities:
+- Members triage the project’s issues and pull requests.
+- Member maintain and monitor the website including updating versions.
+- Mentor new contributors to the website.
+- Run or support djangoproject.com sessions in DjangoCon sprints.
+- Chair, Co-Chair and Board Liaison can sign off on new features.
+
+## Initial membership
+
+- Chair: Sarah Abderemane
+- Co-Chair: Saptak Sengupta
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): Sarah Abderemane
+- Other members:
+    - Eric Sherman
+    - Mark Walker
+    - Jason Judkins
+    - Paolo Melchiorre
+    - Sanyam Khurana
+    - Tobias McNulty
+    - Ron Maravanyika
+
+
+
+## Future membership
+
+### Who is eligible to join? Any volunteer, or are there specific requirements?
+
+Members must have interest in Django and should be able to work with Django. Members must be well versed with the process of contributing to **djangoproject.com** or at least willing to be guide. We welcome all experience levels, we also welcome first time contributors. 
+
+### How do people who want to join sign up / volunteer / express interest?
+Individuals can express interest by emailing to the working group mailing list at `website-wg@djangoproject.com` 
+
+### How will decisions on adding/removing members be handled?
+Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members. The WG will vote for New Chair/Co-Chairs and decision to appoint will be based on gaining majority votes.
```

Comment:
```suggestion
Direct membership: new members may self-nominate opening a PR to this repository; the WG members will approve the PR (with a required quorum of 50% + 1) to approve new members, or comment the PR with motivation to deny, having at maximum 1 month to express their intent. The WG members will vote for New Chair/Co-Chairs and decision to appoint will be based on gaining majority votes, using the same PR flow.
```
As I said above , let's use one tool for everything.

Reply:
Suggestion: have a smaller group be responsible for approving members - maybe just the chair and co-chair? Or them plus a couple of others?

Background is that the way DSF membership works right now involves a full board vote, similar to what you're proposing here, and it's a pain honestly. It takes a lot of time, and it means there's always a pretty long delay between application and approval. And, most of the time, it's pretty obvious, there are very very few non-obvious decisions. So I'd suggest you aim to streamline joining the group, with just a few people delegated to make quick decisions. You could always toss any difficult calls to the broader membership, so only the complex ones need take longer.

Reply:
> Suggestion: have a smaller group be responsible for approving members - maybe just the chair and co-chair? Or them plus a couple of others?

Great suggestion. I updated my proposal of changes.